src/main/java/commands/CommandResponse.java

```diff
@@ -0,0 +1,27 @@
+package commands;
+
+public class CommandResponse {
+    private String responseMsg;
+    private boolean isExit;
+
+    public CommandResponse(String responseMsg, boolean isExit){
+        this.responseMsg = responseMsg;
+        this.isExit = isExit;
+    }
+
+    public String getResponseMsg() {
+        return responseMsg;
+    }
+
+    public void setResponseMsg(String responseMsg) {
```

Comment:
Great namings for all these methods, really shows what does each method do

Reply:
Thanks!